contents/careers.md

```diff
@@ -4,30 +4,70 @@ sidebar: null
 
-This is a super exciting time to be joining PostHog. We're growing like crazy and we need your help.
+Our GitHub repo has over over 2.8k stars (and growing fast), and we already have thousands of users, including deployments at large companies. 
 
-We are _fully-remote_, and actively are seeking a diverse and multinational team.
+We're backed by some of the world's top investors and believe in product-led growth, where we build something awesome and let our product bring the users, rather than a sales team and regular cold calls.
 
-## Our culture
+And last but not least, we're growing like crazy and need your help. It's a very exciting time to be joining PostHog.
 
-In reality, as one of our first hires you'll have a large hand in shaping this, but some keywords:
+## Our Culture
 
-* Remote first
-* Strong written communication -- most of our communication happens over GitHub 
-* Prefer making things transparent and public
-* No need for crazy hours. Work hard when you're working.
+As a young company, our culture is still being polished by our growing team, and, as one of our first hires, you'll 
+play an important role in shaping it. 
 
-## Our backers
+However, there are some things we've already established as our core values:
 
-We're young, but we're already [backed](/handbook/strategy/investors) by many of the best investors in the world and have millions of dollars in funding.
+##### Remote First
 
-This includes YCombinator, Solomon Hykes (ex-founder/CEO Docker), David Cramer (Founder/CEO Sentry), Adam Goldstein (Founder Hipmunk). See [the rest](/handbook/strategy/investors)!
+We are _fully remote_ and hire anywhere in the world. After all, why limit your talent pool? If you're a fit, we want you on board. 
+
+Currently, no two PostHog members live in the same city, not even our co-founders! But don't worry, we'll still hire you even if you do share a city with a team member.
+
+##### Diversity
+
+Being fully remote means we're able to create a team that is _truly_ diverse.
+
+In our view, a non-traditional team is how we build a special company. We believe diversity leads to the best results and encourage applicants from any background or experience to drop us a note. 
+
+Furthermore, to ensure we can support the needs of our diverse team well, we offer unlimited time off (minimum 20 days!), don't track hours (work hard when you work), and [offer generous parental leave](/handbook/people/time-off#parental-leave) independent of gender. 
+
+##### Transparency
+
+As the builders of an open-source product, we believe it is only right that we be as transparent as possible as a company.
+
+And this isn't just a meaningless corporate statement! Most of our communication happens publicly on GitHub, our roadmap is open for anyone to see, and this website hosts a [comprehensive handbook](/handbook) explaining anything from how we hire and pay employees to how we email investors!
```

Comment:
I think one ! too many

Reply:
Yeah, great point. I actually had even one more and then removed it because I thought that was one too many. Two too many I guess 😂

Reply:
haha